Sources/MapboxNavigation/DayStyle.swift

```diff
@@ -248,6 +248,10 @@ open class DayStyle: Style {
             UITableView.appearance(whenContainedInInstancesOf: [StepsViewController.self]).sectionHeaderTopPadding = 0.0
         }
         #endif
+        
+        EndOfRouteCommentView.appearance().backgroundColor = #colorLiteral(red: 1, green: 1, blue: 1, alpha: 1)
```

Comment:
There are other `EndOfRoute*` parts above, should we move these lines there?

Reply:
Yep, makes sense, will move.